src/sentry/static/sentry/app/views/organizationDashboard.jsx

```diff
@@ -207,6 +209,125 @@ const Activity = React.createClass({
   },
 });
 
+const EPH = React.createClass({
+
+  mixins: [ApiMixin, OrganizationState],
+
+  getInitialState() {
+    let until = Math.floor(new Date().getTime() / 1000);
+    let since = until - 3600 * 24;
+
+    return {
+      rawOrgData: {},
+      formattedData: null,
+      querySince: since,
+      queryUntil: until,
+      error: false
+    };
+  },
+
+  componentDidMount() {
+    this.fetchData();
+  },
+
+  // Use this so there's a standard order for y values and bar classes
+  STAT_OPTS: ['received', 'rejected', 'blacklisted'],
+
+  fetchData() {
+    let statEndpoint = this.getEndpoint();
+
+    let query = {
+      since: this.state.querySince,
+      until: this.state.queryUntil,
+      resolution: '1h',
+    };
+
+    $.when.apply($, this.STAT_OPTS.map(stat => {
+        let deferred = $.Deferred();
+        this.api.request(statEndpoint, {
+          query: Object.assign({stat: stat}, query),
+          success: deferred.resolve.bind(deferred),
+          error: deferred.reject.bind(deferred)
+        });
+        return deferred;
```

Comment:
It would be great to improve our "api" helper to work with promises without having to do this.